iothub/device/samples/convention-based-samples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -59,7 +59,7 @@ public async Task PerformOperationsAsync(CancellationToken cancellationToken)
 
             // Verify if the device has previously reported the current value for property "serialNumber".
             // If the expected value has not been previously reported then send device serial number over property update.
-            // This is a root-level property update call.
+            // This is a top-level property update call.
```

Comment:
renamed all occurances of "root-level" to "top-level" => since this is the term that our PnP docs use: https://docs.microsoft.com/en-us/azure/iot-pnp/concepts-convention